conan/tools/files/files.py

```diff
@@ -94,7 +94,7 @@ def rm(conanfile, pattern, folder, recursive=False, excludes=None):
 
 def get(conanfile, url, md5=None, sha1=None, sha256=None, destination=".", filename="",
         keep_permissions=False, pattern=None, verify=True, retry=None, retry_wait=None,
-        auth=None, headers=None, strip_root=False, extract_filter=None):
+        auth=None, headers=None, strip_root=False, extract_filter=None, exclude_pattern=None):
```

Comment:
``exclude_pattern`` name is a bit misleading because of being singular, as it will be a list, better be plural? this is why the ``copy(..., excludes=[...])`` is ``excludes``